src/commands/dev/lib/dev-server.ts

```diff
@@ -334,8 +334,14 @@ export default class DevServer {
 
       case 'Lambda':
         if (!asset.fn) {
-          res.statusCode = 500;
-          res.end('Lambda function has not been built');
+          // This is mostly to appease TypeScript since `fn` is an optional prop,
```

Comment:
Is it possible for a `BuiltLambda` without `fn`?

Reply:
Not really, but if there's a bug that makes this happen somehow then this check will catch it and return the proper error.